### PR TITLE
fix: removed unused files after nemesis block is generated.

### PR DESCRIPTION
**Pull request by fboucquez**
This PR removes the nemesis block temp files after generation. We can then have deterministic nemesis seed folder generation and compare folders in unit tests (Future PR).

### CODE REVIEW

**Comment by Jaguar0625** [comment]
> This PR removes the nemesis block temp files after generation. 

Removes from where? Local machine or a deployed server?

**Comment by fboucquez** [comment]
Hi @Jaguar0625 , the issue description wasn't the best. 

When user runs -p bootstrap, the nemgen creates all these files:

image

This PR is to clean up the files the catapult client doesn't require saving space and helping me with the unit test suite that it's coming

**Comment by Jaguar0625** [comment]
why are you not running nemgen with `useTemporaryCacheDatabase` option?

**Comment by fboucquez** [comment]
> why are you not running nemgen with `useTemporaryCacheDatabase` option?

Thanks! I didn't know the option.

File: src/service/NemgenService.ts [hunk 1]
```diff
@@ -50,24 +50,38 @@ export class NemgenService {
             '--resources=/server-config',
             '--nemesisProperties=./server-config/block-properties-file.properties',
         ];
+        if (presetData.nemgenUseTemporaryCacheDatabase) {
+            cmd.push(`--useTemporaryCacheDatabase`);
+        }
```
Comment (Jaguar0625) [inline src/service/NemgenService.ts:55]: why did you add option for this? when would you want to run WITHOUT it?

Comment (fboucquez) [reply src/service/NemgenService.ts:55]: Only for testing? By providing: `nemgenUseTemporaryCacheDatabase: true` on custom preset.

Comment (fboucquez) [reply src/service/NemgenService.ts:55]: I have never "read" the cache database, if you think nobody will, I can remove the option and always use temp

Comment (Jaguar0625) [reply src/service/NemgenService.ts:55]: i think it is always acceptable to use this option. but you can check with @gimre-xymcity.

Comment (fboucquez) [reply src/service/NemgenService.ts:55]: @gimre-xymcity would somebody ever use the cache database for debugging?

Comment (gimre-xymcity) [reply src/service/NemgenService.ts:55]: nope

**Review by Jaguar0625 — APPROVED** [review]

File: src/service/NemgenService.ts (hunk 1 above)
Comment (fboucquez) [reply src/service/NemgenService.ts]: Removed the option
